MetaMask/metamask-extension
# [Bug]: Enabling the display of token total in SetApprovalForAll

## Explanation
Fixes: #18618 

Problem:
 Whenever we trigger setApprovalForAll function for an ERC721 token, we can see how the Your NFT mayb be at risk warning does not display anymore the Total number of tokens we are approving.

Explanation:
`isERC721`, and the `total` was coming up as false and 0, disabling the Total display in the SetApproveForAllWarning.
 isERC721 was coming up as false because instead of comparing `assetStandard` to ERC721, we were comparing it to ERC20. The total was coming up as 0 because of the recent change to the  `fetchTokenBalance`, which changed the format of `tokenBalance` fetched.

## Screenshots

### Before

<img width="362" alt="Screenshot 2023-04-27 at 1 12 30 PM" src="https://user-images.githubusercontent.com/43930900/234938584-6668ddd6-47b9-4f77-964d-8cd3318dc529.png">

### After

<img width="357" alt="Screenshot 2023-04-27 at 1 11 09 PM" src="https://user-images.githubusercontent.com/43930900/234938280-60e7a7f0-fd83-4386-963a-5aa5435e2027.png">

## Manual Testing Steps

1. Go to test dapp
2. Deploy Collectibles contract
3. Mint some tokens
4. Click Set Approval for All
5. Click Confirm
7. Make sure the Total is now displayed in the pop-up warning.
